ai-missions/high-risk-flu-shot-mission/skills/send_schedule_appointment_notification/Readme.md

```diff
@@ -0,0 +1,36 @@
+### Send Schedule Appointment Notification
+
+
+#### Description:
+For the given profile IDs, this skill sends notifications on preferred communication channel to schedule a telehealth appointment.
```

Comment:
Change to: For the Profile IDs derived from the configured Cohort, notifications are sent on the preferred communication channel that request that members schedule a telehealth appointment.